### PR TITLE
Update views.* endpoints to use POST method instead of GET

### LINKED ISSUE #111: Request-URI Too Large
Some API methods such as views.publish require a large payload and as a result - can fail with `Request-URI Too Large`. Is there a way to have the client send as HTTP POST with either form values or JSON encoded body? Slack API supports both of these options. [Using the Slack Web API](https://api.slack.com/web)

### PR DESCRIPTION
**Pull request by matthewnessworthy**
Fixes #111 
Replaces #130 (because of merge conflicts)

First updated the SDK with the latest updates
```
rm -Rf generated/*
vendor/bin/jane-openapi generate --config-file=.jane-openapi.php
./bin/slack-api-client-generator spec:generate-patch
./bin/slack-api-client-generator spec:update
```

Then after updating the relevant HTTP methods in `resources/slack-openapi-patched.json`
```
./bin/slack-api-client-generator spec:generate-patch
./bin/slack-api-client-generator spec:update
vendor/bin/jane-openapi generate --config-file=.jane-openapi.php
```

### CODE REVIEW

**Comment by matthewnessworthy** [comment]
@damienalexandre please review

File: resources/slack-openapi-patched.json [hunk 1]
```diff
@@ -4237,6 +4237,7 @@
                         "description": "Authentication token. Requires scope: `admin.conversations:write`",
                         "in": "formData",
                         "name": "token",
+                        "required": true,
```
Comment (damienalexandre) [inline resources/slack-openapi-patched.json:4240]: Same comment as the other PR, this has nothing to do here IMO.

**Review by damienalexandre — CHANGES_REQUESTED** [review]

File: resources/slack-openapi-patched.json (hunk 1 above)
Comment (matthewnessworthy) [reply resources/slack-openapi-patched.json]: fixed

**Comment by damienalexandre** [comment]
#147 has been merged, you have to rebuild this PR from "main". Sorry about that :yum:

**Comment by matthewnessworthy** [comment]
@damienalexandre no worries, all done

**Review by damienalexandre — CHANGES_REQUESTED** [review]
The method on viewPublish/viewPush/viewUpdate is still "GET" but the parameters have been changed to "form parameter", am I missing something? viewOpen looks good.

**Review by damienalexandre — APPROVED** [review]

**Comment by matthewnessworthy** [comment]
Glad i could help, thanks for the support and hand-holding!

**Pull request merged**